proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -765,12 +784,17 @@ message RangeJoinTablesRequest {
   TableReference left_id = 2;
   TableReference right_id = 3;
   repeated string exact_match_columns = 4;
+  // Provide detailed range match parameters for the range join (alternative to providing `range_match`)
   string left_start_column = 5;
   RangeStartRule range_start_rule = 6;
   string right_range_column = 7;
   RangeEndRule range_end_rule = 8;
   string left_end_column = 9;
   repeated Aggregation aggregations = 10;
+  // Specifies the range match parameters as a parseable string. Providing `range_match` in the GRPC call is the
+  // alternative to detailed range match parameters provided in the `range_start_column`, `range_start_rule`,
+  // `right_range_column`, `range_end_rule`, and `left_end_column` fields.
+  string range_match = 11;
```

Comment:
Do we want to expose this to `io.deephaven.qst.table.RangeJoinTable`? (The answer may be no.)

Reply:
I don't think so, this adds a shortcut / additional way to populate the `RangeJoinMatch` but doesn't replace it.